index.html

```diff
@@ -33,6 +33,131 @@ <h1>Stepan Leonov</h1>
         </button>
       </section>
       <div id="bg-image-2"></div>
+      <section class="projects">
+        <h2>Projects</h2>
+        <section class="project-list">
+          <div class="project">
+            <img
+              class="project-img"
+              src="src/index.png"
+              alt="project preview img"
+            />
```

Comment:
## OPTIONAL
- [ ] It would be nice to add the images from the Figma design to make your design look like the original design till you start adding pictures of personal projects